src/utils/connect.js

```diff
@@ -34,7 +34,7 @@ type ConnectOptions = {
   areStatePropsEqual?: boolean,
   areMergedPropsEqual?: boolean,
   storeKey?: boolean,
-  withRef?: boolean,
+  forwardRef?: boolean,
```

Comment:
Doing this change seemed to have no effect on my flow-checker, so I am not sure what this change really does. But it should be more correct at least.